src/js/base/model.js

```diff
@@ -105,7 +105,22 @@ export default Backbone.Model.extend(extend({
 
     return isFunction(messages[category]) ? messages[category] : this[messages[category]];
   },
-  handleMessage({ category, resource, author, payload }) {
+  addMessageToQueue(data) {
+    if (!this.notifications) {
+      this.notifications = [data];
+      return;
+    };
```

Comment:
Is there a way to set this automatically set `model.notifications: []` initially for each model? Do we want to do that?

If we did that we wouldn't have to check if `this.notifications` exists here. And just do the `push()`.

Reply:
And wasn't sure what to use for naming. But went with `model.notifications` for now.